library.c

```diff
@@ -1092,9 +1091,9 @@ int redis_cmd_append_sstr_zval(smart_string *str, zval *z, RedisSock *redis_sock
         retval = redis_cmd_append_sstr(str, val, vallen);
         if (valfree) efree(val);
     } else {
-        zstr = zval_get_string(z);
+        zstr = zval_get_tmp_string(z, &tmp_str);
         retval = redis_cmd_append_sstr_zstr(str, zstr);
-        zend_string_release(zstr);
+        zend_tmp_string_release(tmp_str);
```

Comment:
I guess this change is done in #2578 so lets remove it from this PR to make it clean